### ISSUE TITLE
Explore wrapup

### ISSUE THREAD

**Issue by abettermap** [Language-Mapping/language-map#128]
## Source
__Originally posted by @rperlin-ela in https://github.com/Language-Mapping/language-map/pull/127#issuecomment-723375179__

### Bugs
* [x] Support commas in URLs, e.g. `/Explore/Macrocommunity/Shi'a,%20Sunni`. **I am no longer seeing this comma'ed instance, only Shi'a and Sunni separately, so calling it resolved.**
* [x] Correct Vatican City flag (see #130)

### Panels/breadcrumbs
* [x] Panel title bar 24/7, sans breadcrumbs in Home (and maybe Details) 
* [x] ...and return the `?` help btn there once again and drop it from the bottom nav.
* [x] and also move the `i` from the map ctrl btns as well.

### Card lists
* [x] can nix the word "Example/s", but maybe just add ellipsis as elsewhere, or ~~ideally at least where there's more to show~~ **ideally yes, makes the most sense, and i gave it a shot but will have to call wishlist on this.** so `...` 24/7 for now, better than empty space. will keep an eye out for solution to reach the ideal. Added it to wishlist in #132 
* [x] Card lists: rm card header icons where too redundant, e.g. on every language card in /Explore/Language. Likewise under Language Family and Neighborhood, though not World Region or Country since the colors introduce some variation.

### Size/Status
* [x] Remove from Explore
* [x] ...and make them not clickable in legend

### Blurbs
* [x] ...then update legend wording and only show if World Region is the symb type. Might as well include the "Show world map" toggle as well.
* [x] Add blurb in Explore panels to encourage users to view Data:
  > Yes, agreed, let's use similar language to what's in the table. Like "To view results in the map, filter columns in the **Data** table" or something like that?

**Comment by abettermap** [comment]
Ross- i think my edits above cover what's left of what we discussed. feel free to add/correct if i missed any!

**Comment by abettermap** [comment]
New members of the panel title bar party are getting pretty snug but I think I played it well with the transparent green shadow overlapping the overflowed crumbs:

![image](https://user-images.githubusercontent.com/4974087/98633179-255f0380-22de-11eb-8763-7abce75e52ca.jpeg)

I tried ellipsis instead of shadow and it was fine but only for the very very last depths. Sometimes the level above that was so long that the crumb would wrap, but I didn't want to pursue ellipsizing that one as that was another level of difficulty. 

I think the transparent shadow looks better anyway, especially considering the final depth crumb is often the title of the current view already. Aka pretty redundant and expendable. 

Obviously the home icon is not useful on the home view, but the consistency with the other views felt better than "blank empty crumb space on home but nowhere else". 

I think the only outstanding thing I have left with crumbs is to show /Details/LanguageName instead of /Details/ID

The right side bar icons are pretty snug but no to the point where fat-fingerers would struggle. Ok I'm not one of them so let's get some chubby-digited clumsies up in here for testing. 🤙🏼
